lib/stm32/h7/Makefile

```diff
@@ -0,0 +1,52 @@
+##
+## This file is part of the libopencm3 project.
+##
+## This library is free software: you can redistribute it and/or modify
+## it under the terms of the GNU Lesser General Public License as published by
+## the Free Software Foundation, either version 3 of the License, or
+## (at your option) any later version.
+##
+## This library is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+## GNU Lesser General Public License for more details.
+##
+## You should have received a copy of the GNU Lesser General Public License
+## along with this library.  If not, see <http://www.gnu.org/licenses/>.
+##
+
+LIBNAME		= libopencm3_stm32h7
+SRCLIBDIR	?= ../..
+
+CC		= $(PREFIX)gcc
+AR		= $(PREFIX)ar
+
+# STM32H7 supports double precision FPU
+FP_FLAGS	?= -mfloat-abi=hard -mfpu=fpv5-d16
```

Comment:
does h7 have both single and double like the f7 has done? or don't we have to worry about that mess?

Reply:
To date, I'm unaware of any single precision H7 variant, but that doesn't mean they won't make one.

Reply:
that's at least more promising than f7 where we already have both :)